src/views/usages/UsagesTree/UsagesTreeProvider.ts

```diff
@@ -82,8 +82,8 @@ export class UsagesTreeProvider
         const variablesCLIController = new VariablesCLIController(folder)
 
         let variables: Record<string, Variable> | undefined = undefined
-        // when current org is changed, project is undefined
-        if (StateManager.getFolderState(folder.name, KEYS.PROJECTS)) {
+        // when the current organization is changed in the Home View, current project is undefined until a new one is selected
+        if (StateManager.getFolderState(folder.name, KEYS.PROJECT_ID)) {
```

Comment:
it seems like this check should actually be within `getAllVariables`? since anytime that's called when the project isn't set we shouldn't call the CLI?

Reply:
good call. Will make the change